### PR TITLE
Created the press page.

### LINKED ISSUE #139: Add a media kit
This is how the site could look like
Bildschirmfoto 2019-06-06 um 08 54 45

It would cool to add a hover effect to visualize the possibility to download each file

File:
[Media_kit2.pdf](https://github.com/gitpod-io/website/files/3210908/Media_kit2.pdf)

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #139 
image
image

### CODE REVIEW

File: src/resources/logo-black.svg [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="logo-final" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 632.2 206.4" style="enable-background:new 0 0 632.2 206.4;" xml:space="preserve">
```
Comment (jankeromnes) [inline src/resources/logo-black.svg]: Please use https://jakearchibald.github.io/svgomg/ or https://github.com/svg/svgo on all the SVG logos to optimize their source (it's currently made heavy by many useless attributes).

File: src/pages/press.tsx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+                        <a href={mediaKitPDF} download>
+                            <img src={mediaKitPreview} alt="Media Kit PDF Preview" className="preview-img" />
+                        </a>
+                        <p style={{ marginTop: 15 }}>Fact Sheet</p>
+                    </div>
+                    <div>
+                        <div className="logo">
+                            <div className="svg-container" style={{ background: '#fff' }}>
+                                <a download href={LogoWhite}>
+                                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 632.19 206.35">
```
Comment (jankeromnes) [inline src/pages/press.tsx]: Please don't copy/paste the entire SVG markup. Instead, you can use `import` to import and use the SVG logo (please search for other SVG imports in the website for an example, e.g. by running `git grep import | grep svg`).

File: src/pages/press.tsx [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                <Logos logos={[
+                    [1050, 125, 40],
+                ]} />
+                <div style={{ marginTop: 60 }}>
+                    <h4 style={{ color: colors.fontColor2 }}>Media kit</h4>
+                    <h1>Press Information</h1>
+                    <UnderLine />
+                </div>
+                <p style={{ marginBottom: 80 }} >
+                    Would you like to cooperate with us? Write something about Gitpod? Or maybe you are just curious?<br />Our media kit will give an overview about Gitpod.
```
Comment (jankeromnes) [inline src/pages/press.tsx]: This text is a bit strange. Here is a suggestion to improve it:

```suggestion
                    Are you looking to write a story about Gitpod? We'd love to help you with that!<br />Here are our official logos and a quick fact sheet about Gitpod.
```

File: src/pages/press.tsx [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+                </p>
+                <MediaKit>
+                    <div style={{ position: 'relative' }}>
+                        <a href={mediaKitPDF} download>
+                            <img src={mediaKitPreview} alt="Media Kit PDF Preview" className="preview-img" />
```
Comment (jankeromnes) [inline src/pages/press.tsx]: I think that technically, the "media kit" is this entire page (i.e. the "fact sheet PDF" + the logos).

```suggestion
                            <img src={mediaKitPreview} alt="Fact Sheet PDF Preview" className="preview-img" />
```

File: src/pages/press.tsx [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+                                        <polygon className="st2" fill="#0092CF" points="89.3,103.2 89.3,103.2 89.3,103.2 143,72.2 143,72.2"
+                                        />
+                                        <polygon className="st2" fill="#0092CF" points="161,61.8 161,61.8 178.7,51.6 178.7,51.6"
+                                        />
+                                        <polygon className="st2" fill="#0092CF" points="178.7,51.6 161,61.8 143,72.2 89.3,103.2 89.3,103.2 89.3,103.2 89.3,103.2 89.3,112.4 89.3,112.4 89.3,136.9 89.3,136.9 89.3,164.9 89.3,164.9 143,134.1 143,113.7 107,134.2 107,113.4 161,82.7 161,144.6 89.3,185.6 89.3,206.4 178.7,154.8 178.7,154.8 178.7,154.8"
+                                        />
+                                    </svg>
+                                </a>
+                            </div>
+                            <span>Figurative logos</span>
```
Comment (jankeromnes) [inline src/pages/press.tsx]: I only see one logo here. Also, I don't quite understand "figurative" here. Here is an alternative suggestion:

```suggestion
                            <span>Logo without text</span>
```

File: src/pages/press.tsx [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+                        </div>
+                    </div>
+                </MediaKit>
+                <p>If you have any question or feedback, please contact us.&nbsp;&nbsp;&nbsp;&nbsp;
+                    <a href='#'>
```
Comment (jankeromnes) [inline src/pages/press.tsx]:
```suggestion
                    <a href="mailto:contact@gitpod.io">
```

File: src/resources/logo-black.svg (hunk 1 above)
Comment (nisarhassan12) [reply src/resources/logo-black.svg]: @jankeromnes Thanks! for the suggesstion. I will always do that in future.

**Pull request merged**